packages/esm-patient-chart-app/src/visit/visit-prompt/start-visit-dialog.component.tsx

```diff
@@ -1,37 +1,33 @@
 import React, { useCallback } from 'react';
 import { useTranslation } from 'react-i18next';
-import { ComposedModal, Button, ModalBody, ModalHeader, ModalFooter } from 'carbon-components-react';
+import { Button, ModalBody, ModalHeader, ModalFooter } from 'carbon-components-react';
 import { launchPatientWorkspace } from '@openmrs/esm-patient-common-lib';
 import styles from './start-visit-dialog.scss';
-import { useVisitDialog } from '../useVisitDialog';
 
 interface StartVisitDialogProps {
   patientUuid: string;
+  closeModal: () => void;
+  visitType: string;
```

Comment:
@donaldkibet Hey, I know this is from a long time ago, but where is this `visitType` supposed to come from? I don't see how it could possibly get this param passed to it, the way it's presently used in the application. I also don't understand the use case.